Added the documentation

SiddharthaArepalli opened the pull request:
Added the minimal  documentation page which was not present early in the project 
![Screenshot 2024-10-11 095853](https://github.com/user-attachments/assets/791b8eab-012c-427e-bd51-ccae6c1edeab)

harshithtunuguntla commented:
Hi @SiddharthaArepalli ,

Great that you've worked out on the documentation page. However, I've noticed a few mistakes in the information being provided on the webpage for documentation. Attached are the screenshots.

We don't have these features as of now.
<img width="542" alt="image" src="https://github.com/user-attachments/assets/2210fbcd-db0c-4618-a965-80b3f86415b4">

Same goes with this FAQ, we don't have any as of now.
<img width="681" alt="image" src="https://github.com/user-attachments/assets/dcfa7af3-1d08-42e0-9085-403cbcb11057">

And the details mentioned here are wrong too, they are not leading anywhere along with email not correct, whatsapp and instagram not having any hyperlinks
<img width="683" alt="image" src="https://github.com/user-attachments/assets/1db57929-e9ee-44c1-98f7-3edc9f4c07b2">

We can do two things here, please let me know which one you'd like to go ahead with.
- I can approve the PR since these are information changes and you or someone else can pick these up as a different PR (however, this is not an ideal way that happens in organisations)
- You can make these changes and simply commit, push them to the same PR.

Do let me know which one do you wanna go with, as I said, this is a wonderful jobs, thanks for contributing!

SiddharthaArepalli commented:
Hie @harshithtunuguntla , i have added that information keeping in mind of features that will added in future any ways i have removed the info which is not relavant to the project as per now , and i made a new PR with the changes do check it out !!
Thanks for the feedback !!
